openpgp/read_test.go

```diff
@@ -800,6 +800,7 @@ func TestSymmetricAeadEaxOpenPGPJsMessage(t *testing.T) {
 func TestCorruptedMessageInvalidSigHeader(t *testing.T) {
 	// Decrypt message with corrupted MDC and invalid one-pass-signature header
 	// Expect parsing errors over unverified decrypted data to be opaque
+	var expectedErr string = errors.DecryptWithSessionKeyError(errors.GenericParsingErrorMessage).Error()
```

Comment:
Minor nitpick but I'm in general not a huge fan of sharing constants between code and tests, especially if it's the thing we're testing.
In general if we want to change the error message, I think it's perfectly fine to have to change both the code and the tests; in a way it highlights that we're changing the observable behavior (even if it's in a minor way).

Reply:
changed with: https://github.com/ProtonMail/go-crypto/pull/237/commits/c6358f155891e94c0a22d27ed0482722b32753b9